docs/reference/index-modules/translog.asciidoc

```diff
@@ -88,59 +88,4 @@ file based sync. Defaults to `512mb`
 The maximum duration for which translog files will be kept. Defaults to `12h`.
 
 
-[float]
-[[corrupt-translog-truncation]]
-=== What to do if the translog becomes corrupted?
-
```

Comment:
To avoid having to set up redirects & help steer people to the new tool, I'd keep this section heading and just xref the new one.